compiler/parser/ceda/ArgCharStack2.c

```diff
@@ -0,0 +1,125 @@
+#include <string.h>
```

Comment:
Is there an ArgCharStack1.c or ArgCharStack.c? I'm even more confused...

Reply:
@thurston ?

Reply:
This is part of the C port. It does not affect the Python port. The change is appearing now because of the vagaries of git branching.

Somewhere in the ceda branch of git, there is ArgCharStack.c, which used a linked list (the "natural" translation of the OCaml list code). This optimized version uses an expanding array instead.